Src/WitsmlExplorer.Frontend/contexts/queryContext.tsx

```diff
@@ -101,22 +161,34 @@ const retrieveStoredQuery = () => {
     validateQueryState(queryState);
     return queryState;
   } catch {
-    return defaultQueryContextState;
+    return getDefaultQueryContextState();
   }
 };
 
 const setStoredQuery = (queryState: QueryContextState) => {
```

Comment:
That's the way it's done most places in WitsmlExplorer. Some places, it's not even wrapped in a try/catch. I don't know if WitsmlExplorer is supposed to be supported in browsers that doesn't have a localStorage.

Reply:
I think we should at least consider adding a warning message to the console to provide feedback. This will help future developers to debug the feature more easily if they face any issues related to data not being added to the localStorage.